index.js

```diff
@@ -89,7 +89,7 @@ var Sqlite3Driver = Base.extend({
     if (spec.defaultValue !== undefined) {
       constraint.push('DEFAULT');
 
-      if(typeof(spec.defaultValue) === 'string')
+      if(typeof(spec.defaultValue) === 'string' && spec.defaultValue.indexOf('(') === -1)
```

Comment:
@wzrdtales I think it probably gets a bit hard to determine whether an arbitrary string is actually a valid function.  Maybe I should extend `defaultValue` to allow an object like this;

~~~
{ value: 'someString or function', quote: false }
~~~

Where `quote` would default to `true`?